metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/query/request/SearchRequestHandler.java

```diff
@@ -247,14 +249,16 @@ private List<AggregationBuilder> getAggregations() {
 
   private HighlightBuilder getHighlights() {
     HighlightBuilder highlightBuilder = new HighlightBuilder();
+
     // Don't set tags to get the original field value
     highlightBuilder.preTags("");
     highlightBuilder.postTags("");
+
     // Check for each field name and any subfields
-    _defaultQueryFieldNames.forEach(fieldName -> highlightBuilder
-            .field(fieldName)
-            .field(fieldName + ".*"));
-    highlightBuilder.field("urn.delimited");
+    _defaultQueryFieldNames.stream()
+            .flatMap(fieldName -> Stream.of(fieldName, fieldName + ".*")).distinct()
+            .forEach(highlightBuilder::field);
```

Comment:
Did we verify the column name highlighting still works after this?

Reply:
Yes